src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
+import { Navbar,Footer } from './components';
+import { Landing ,Rules,Signup,Login} from './pages';
 import './App.css';
+import {Routes,Route} from "react-router-dom"
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Landing />} />
+        <Route path="/rules" element={<Rules />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route path="/login" element={<Login />} />
```

Comment:
You can ommit the usage of `/` as it's not reqd in react router v6

Reply:
Okay